### PR TITLE
Add disk cache removal to UI

### PR DESCRIPTION
**Pull request by MSuih**
Adds a more convinient and user-friendly way of removing disk cache; by just clicking on `File > All Games > Remove Disc Cache`.

### CODE REVIEW

File: rpcs3/rpcs3qt/main_window.cpp [hunk 1]
```diff
@@ -1703,6 +1705,21 @@ void main_window::SetIconSizeActions(int idx)
 		ui->setIconSizeLargeAct->setChecked(true);
 }
 
+
+void main_window::RemoveDiskCache()
+{
+	std::string cacheDir = Emulator::GetHdd1Dir();
+	if (fs::exists(cacheDir) && fs::is_dir(cacheDir))
```
Comment (elad335) [inline rpcs3/rpcs3qt/main_window.cpp]: is_dir evalutes to false if doesn't exist.

File: rpcs3/rpcs3qt/main_window.cpp [hunk 2]
```diff
@@ -1252,6 +1252,8 @@ void main_window::CreateConnects()
 	connect(ui->batchRemoveCustomConfigurationsAct, &QAction::triggered, m_gameListFrame, &game_list_frame::BatchRemoveCustomConfigurations);
 	connect(ui->batchRemoveCustomPadConfigurationsAct, &QAction::triggered, m_gameListFrame, &game_list_frame::BatchRemoveCustomPadConfigurations);
 
+	connect(ui->removeDiskCacheAct, &QAction::triggered, [this] {RemoveDiskCache(); });
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/main_window.cpp]: There is no need to use a lambda if you're already using a qslot

File: rpcs3/rpcs3qt/main_window.cpp [hunk 3]
```diff
@@ -1703,6 +1705,21 @@ void main_window::SetIconSizeActions(int idx)
...
 
+
+void main_window::RemoveDiskCache()
+{
+	std::string cacheDir = Emulator::GetHdd1Dir();
+	if (fs::exists(cacheDir))
+	{
+		if (!fs::remove_all(cacheDir, false))
+		{
+			QMessageBox::warning(this, tr("Error"), tr("Could not remove cache"));
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/main_window.cpp]: Might as well call it disk cache...
Same below

File: rpcs3/rpcs3qt/main_window.cpp [hunk 4]
```diff
@@ -1637,6 +1639,20 @@ void main_window::SetIconSizeActions(int idx)
 		ui->setIconSizeLargeAct->setChecked(true);
 }
 
+void main_window::RemoveDiskCache()
+{
+	std::string cacheDir = Emulator::GetHdd1Dir();
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/main_window.cpp]: cache in system.cpp is
`const std::string cache_location = Emulator::GetHdd1Dir() + "/cache";`
Why the change here?

Comment (MSuih) [reply rpcs3/rpcs3qt/main_window.cpp]: Cache folder is easier to iterate through for the limiter. This one just deletes everything so I might as well just delete absolutely everything.

Most data is stored in the cache folder but I've seen some dangling files get left to the root of the drive or other folders in it. I'm not sure what the logic is there.

Comment (MSuih) [reply rpcs3/rpcs3qt/main_window.cpp]: Changed as requested
